CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
```

Comment:
Is it on by default on AOT then?

How do I turn it on?

Would be nice to have a mention of that in the changelog too.

Reply:
Perhaps something we could also document in `Troubleshooting` when released?

Reply:
Yeah when @Flash0ver and I discussed we were thinking we wouldn't promote this too widely, since it's quite experimental. However there are some additional docs on `SentryOptions.UseStackTraceFactory` (which is how you'd add this):
https://github.com/getsentry/sentry-dotnet/blob/e4120a976a1067c568fe6060cf3ad8158d6c44e6/src/Sentry/SentryOptions.cs#L1637-L1643

And we figured we'd add something to our [troubleshooting docs](https://docs.sentry.io/platforms/dotnet/troubleshooting/) as well... 

I'll add a blurb in the description of this PR as well.